src/main/java/io/github/oliviercailloux/y2018/apartments/apartment/Apartment.java

```diff
@@ -253,16 +250,16 @@ public int getNbBathrooms() {
 	}
 
 	/**
-	 * get the presence (or absence) of a terrace
+	 * Gets the presence (or absence) of a terrace
 	 * 
-	 * @return a boolean
+	 * @return a Boolean
 	 */
 	public Boolean getTerrace() {
```

Comment:
Non, `boolean`.

Reply:
> Bonjour, nous avions changé cela lorsque nous avons rendu les critère booléens comme obligatoire. Préférez vous un Optional ? Cela nous permettrais de vérifier si la valeur a bien été initialisée lors de la création de l'objet.

Reply:
> une solution serait de mettre les setters et les getters en boolean mais de laisser l'objet en Boolean au sein de la classe. Est ce que cela vous conviendrait ?

Reply:
Fixed in commit a0d76b2f15b1628af6d8570b06c8c3bee2bddb02